todo_app/data/trello_items.py

```diff
@@ -0,0 +1,136 @@
+# -*- coding: utf-8 -*-
+from telnetlib import STATUS
+from turtle import update
```

Comment:
```suggestion

```
I think these two lines are accidental imports, and we're not actually using them